apps/microtvm/reference-vm/zephyr/base-box/test-config.json

```diff
@@ -1,4 +1,12 @@
-{"vid_hex": "0483",
- "pid_hex": "374b",
- "test_cmd": ["pytest", "tests/micro/qemu/test_zephyr.py", "--microtvm-platforms=stm32f746xx"]
+{
+    "stm32f746xx": {
+        "vid_hex": "0483",
+        "pid_hex": "374b",
+        "test_cmd": ["pytest", "tests/micro/qemu/test_zephyr.py", "--microtvm-platforms=stm32f746xx"]
+        },
+    "nrf5340dk": {
```

Comment:
@mdw-octoml could you just confirm you see these USB vid/pid?

Reply:
Confirmed!